library/testing/TestSDKOptions.cxx

```diff
@@ -106,34 +106,34 @@ int TestSDKOptions(int argc, char* argv[])
     return EXIT_FAILURE;
   }
 
-  // Test int vector
-  opt.set("resolution", { 1000, 600 });
-  if (opt.getAsIntVector("resolution") != std::vector<int>{ 1000, 600 })
-  {
-    std::cerr << "Options getAsIntVector is not behaving as expected." << std::endl;
-    return EXIT_FAILURE;
-  }
-  std::vector<int> valIntVec;
-  opt.get("resolution", valIntVec);
-  if (valIntVec != std::vector<int>{ 1000, 600 })
-  {
-    std::cerr << "Options get int vector is not behaving as expected." << std::endl;
-    return EXIT_FAILURE;
-  }
-  opt.set("resolution", std::vector<int>{ 1000, 700 });
-  if (opt.getAsIntVectorRef("resolution") != std::vector<int>{ 1000, 700 })
-  {
-    std::cerr << "Options set int vector is not behaving as expected." << std::endl;
-    return EXIT_FAILURE;
-  }
-  std::vector<int>& refIntVec = opt.getAsIntVectorRef("resolution");
-  refIntVec = std::vector<int>{ 1000, 800 };
-  opt.get("resolution", valIntVec);
-  if (valIntVec != std::vector<int>{ 1000, 800 })
-  {
-    std::cerr << "Options getAsIntVectorRef is not behaving as expected." << std::endl;
-    return EXIT_FAILURE;
-  }
+  // Test int vector TODO cannot test without an option ?
```

Comment:
Remove this commented code. I'm fine with removing the vec<int> api if you want to.

Reply:
lets keep the API for now, Ill just remote the commented code